frontend/src/containers/RegistrationContainer/RegistrationContainer.js

```diff
@@ -0,0 +1,48 @@
+import Registration from "../../components/Registration/Registration";
+import { ACCOUNTS } from "../../properties/endpoints";
+
+function postRegistrationDetails(registrationDetails) {
+	console.log(registrationDetails);
```

Comment:
Test log? should be removed in production

Reply:
Ooops :(